### ISSUE TITLE
Image Chooser style makes images go to 2 columns when preview panel is open

### ISSUE THREAD

**Issue by bmoe872** [wagtail/wagtail#11785]
### Issue Summary
When the preview panel is open. Going to an image chooser squishes the images to 2 columns. 

### Steps to Reproduce

Go to a page to edit that has a image chooser on it. Open the preview panel. Open the image chooser, and see that the images are squished to 2 columns.

### Technical details

-   Python version: 3.11.1
-   Django version: 5.0.3
-   Wagtail version: 6.0.1

### Working on this

This is due to the two styles being applied .side-panel-open .tab-content 

I was able to get around the issue by using the wagtail hook `insert_global_admin_css` like this:
```
@hooks.register("insert_global_admin_css")
def editor_css():
    # this patches an issue with images going into 2 columns when the preview panel is open.
    return mark_safe(
        """
        <style>
        .side-panel-open .tab-content{
            width: 100%;
        }
        </style>
        """
    )
```

Anyone can contribute to this. View our [contributing guidelines](https://docs.wagtail.org/en/latest/contributing/index.html), add a comment to the issue once you’re ready to start.

**Comment by rd3m** [comment]
Flagging as fixed in 6.0.2 in #11703

**Comment by laymonage** [comment]
Yep, thanks! We'll put out a 6.0.2 release in the next week or so, which will include the fix.
